HISTORY.md

```diff
@@ -1,5 +1,9 @@
 # cloudpathlib Changelog
 
+## v0.5.1 (unreleased)
```

Comment:
I think we want this to be v0.6.0. Per Semver:

> Given a version number MAJOR.MINOR.PATCH, increment the:
> 
> 1. MAJOR version when you make incompatible API changes,
> 2. **MINOR version when you add functionality in a backwards compatible manner**, and
> 3. PATCH version when you make backwards compatible bug fixes.

Also we probably just want to release after this is merged, so you can probably replace the `(unreleased)` with the date and bump the version in `setup.py`.